crypto/src/main/kotlin/web5/sdk/crypto/InMemoryKeyManager.kt

```diff
@@ -87,7 +87,29 @@ public class InMemoryKeyManager : KeyManager {
     keyStore[keyAlias] ?: throw IllegalArgumentException("key with alias $keyAlias not found")
 
   /**
-   * Imports [jwk] and returns the alias that refers to it.
+   * Imports a list of keys represented as a list of maps and returns a list of key aliases referring to them.
+   *
+   * @param keySet A list of key representations in map format.
+   * @return A list of key aliases belonging to the imported keys.
+   */
+  public fun import(keySet: List<Map<String, Any>>): List<String> {
+    val keyAliases = mutableListOf<String>()
+
+    for (jsonJwk in keySet) {
+      val jwk = JWK.parse(jsonJwk)
+      val keyAlias = import(jwk)
+
+      keyAliases.add(keyAlias)
+    }
+
+    return keyAliases
+  }
```

Comment:
Perhaps this might be simpler? 
```suggestion
  public fun import(keySet: Iterable<Map<String, Any>>): List<String> = keySet.map {
    val jwk = JWK.parse(it)
    import(jwk)
  }
```